src/stan-users-guide/measurement-error.Rmd

```diff
@@ -4,12 +4,12 @@ Most quantities used in statistical models arise from measurements.
 Most of these measurements are taken with some error.  When the
 measurement error is small relative to the quantity being measured,
 its effect on a model is usually small.  When measurement error is
-large relative to the quantity being measured, or when  precise
+large relative to the quantity being measured, or when precise
 relations can be estimated being measured quantities, it is useful to
 introduce an explicit model of measurement error.  One kind of
 measurement error is rounding.
 
-Meta-analysis plays out statistically  much like measurement error
+Meta-analysis plays out statistically much like measurement error
```

Comment:
[question]
I'm curious---how'd you find these errors?  Grep?  They wouldn't show up in the rendered versions.

Reply:
My eyes are 20/20 ;)